app/views/providers/capital_assessment_results/show.html.erb

```diff
@@ -4,8 +4,14 @@
       column_width: :full,
     ) do %>
 
-  <div class="interruption-panel">
-    <%= render @result_partial %>
+  <div class="moj-interruption-card">
+    <div class="moj-interruption-card__content">
+      <div class="govuk-grid-row">
+        <div class="govuk-grid-column-two-thirds">
```

Comment:
Is the grid required? If yes, would it be ideal to have it outside the interruption card component?